.github/workflows/beta-release.yml

```diff
@@ -0,0 +1,75 @@
+name: Release (Beta)
+
+on:
+  push:
+    tags:
+      - v*.*.*
+    branches:
+      - fdoxyz/beta-release-workflow
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+
+      - name: Cache Ruby - Bundler
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Cache Gradle
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches/
+          key: cache-clean-gradle-${{ matrix.os }}-${{ matrix.jdk }}
+
+      - name: Cache Gradle Wrapper
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/wrapper/
+          key: cache-clean-wrapper-${{ matrix.os }}-${{ matrix.jdk }}
+
+      - name: Grant Permission to Execute
+        run: chmod +x gradlew
+
+      - name: Install bundle
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+
+      - name: Configure Keystore
+        run: |
+          echo "This will fail because we don't have the secrets configured yet"
+          echo "$ANDROID_KEYSTORE_FILE" > keystore.jks.b64
+          base64 -d -i keystore.jks.b64 > app/keystore.jks
+          echo "storeFile=keystore.jks" >> keystore.properties
```

Comment:
The workflow from which this was based on ([here](https://github.com/PatilShreyas/AndroidFastlaneCICD/blob/master/.github/workflows/releaseBeta.yml)) creates the `keystore.properties` manually, but we can just base64 encode it whole instead if that feels cleaner. In that case we can replace `KEYSTORE_KEY_ALIAS`, `KEYSTORE_KEY_PASSWORD` and `KEYSTORE_STORE_PASSWORD` for a single secret. I don't feel strongly about either approach

Reply:
We should keep it separate. it will be easier to update